zerver/views/realm.py

```diff
@@ -87,7 +90,7 @@ def update_realm(
         return json_error(_("At least one authentication method must be enabled."))
     if (video_chat_provider is not None and
```

Comment:
I didn't remove these 2 because they have converters, and converters and validators are incompatible. Is there any better way of doing this or this is fine?

Reply:
These should just be switched to validators; there's no good reason for them to be converters.